src/plone/restapi/tests/test_atcontent_serializer.py

```diff
@@ -193,3 +194,119 @@ def test_get_is_folderish_in_folder(self):
         obj = serializer()
         self.assertIn("is_folderish", obj)
         self.assertEqual(True, obj["is_folderish"])
+
+    def test_nextprev_alone(self):
+        folder = api.content.create(
+            container=self.portal,
+            type="Folder",
+            title="Folder with items",
+            description="This is a folder with some documents"
+        )
+        doc = api.content.create(
+            container=folder,
+            type="Document",
+            title="Item 1",
+            description="One item alone in the folder"
+        )
+        data = self.serialize(doc)
+        self.assertEqual(data["previous_item"], None)
+        self.assertEqual(data["next_item"], None)
+
+    def test_nextprev_has_prev(self):
+        folder = api.content.create(
+            container=self.portal,
+            type="Folder",
+            title="Folder with items",
+            description="This is a folder with some documents"
+        )
+        api.content.create(
+            container=folder,
+            type="Document",
+            title="Item 1",
+            description="Previous item"
+        )
+        doc = api.content.create(
+            container=folder,
+            type="Document",
+            title="Item 2",
+            description="Current item"
+        )
+        data = self.serialize(doc)
+        previous_item = {
```

Comment:
Declaring a variable for no good reason makes the code harder to read and understand. Please put that into the assertEqual.